### ISSUE TITLE
while do evaluation in the runhook, the eval_metrics calculate each batch and the average the result or calculate after all evaluation dataset?

### ISSUE THREAD

**Issue by satyrswang** [tensorflow/tensorflow#54573]
like auc in eval_metrics. Anybody knows this?Please help

**Comment by tilakrayal** [comment]
@satyrswang ,
Can you please take a look at this link [1](https://stackoverflow.com/questions/51757345/what-is-the-impact-from-choosing-auc-error-logloss-as-eval-metric-for-xgboost-bi), [2](https://www.tensorflow.org/guide/keras/train_and_evaluate) and [3](https://www.tensorflow.org/api_docs/python/tf/keras/metrics) which delivers the information for eval_metrics.It helps.Thanks!
